### PR TITLE
Correct the comment in PSP examples.

### PR DESCRIPTION
**Pull request by gyliu513**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
none
```

### CODE REVIEW

File: examples/podsecuritypolicy/rbac/bindings.yaml [hunk 1]
```diff
@@ -31,7 +31,8 @@ roleRef:
    kind: ClusterRole
    name: restricted-psp-user
 ---
-# edit grants edit role to system:authenticated.
+# edit grants edit role to the groups
+# restricted and privileged.
```
Comment (CaoShuFeng) [inline examples/podsecuritypolicy/rbac/bindings.yaml]: Good catch!
But I think the group names are:
privileged-psp-users and restricted-psp-users

**Review by CaoShuFeng — CHANGES_REQUESTED** [review]

File: examples/podsecuritypolicy/rbac/bindings.yaml (hunk 1 above)
Comment (gyliu513) [inline examples/podsecuritypolicy/rbac/bindings.yaml]: I think the whole file is using `restricted` and `privileged`, it is actually saying the group attribute here but not the group name

Comment (CaoShuFeng) [inline examples/podsecuritypolicy/rbac/bindings.yaml]: LGTM

**Review by pweil- — APPROVED** [review]

**Pull request merged**